exercises/beer-song/src/Example.hs

```diff
@@ -30,5 +30,5 @@ verse n = unlines
   [ capitalize (stanza n) ++ " on the wall, " ++ stanza n ++ "."
   , action n ++ ", " ++ stanza (pred n) ++ " on the wall." ]
 
-sing :: Int -> Int -> String
-sing from to = unlines $ map verse [from, from - 1 .. to]
+sing :: String
+sing = init $ unlines $ map verse [99, 98 .. 0]
```

Comment:
we were inconsistent! Sometimes using `init` and `unlines` (https://github.com/exercism/xhaskell/blob/master/exercises/food-chain/src/Example.hs#L18), sometimes using `intercalate "\n"` (https://github.com/exercism/xhaskell/blob/master/exercises/house/src/Example.hs#L6)!

I don't actually care though.